crates/red_knot_python_semantic/resources/mdtest/annotations/unsupported_type_qualifiers.md

```diff
@@ -6,14 +6,11 @@ Several type qualifiers are unsupported by red-knot currently. However, we also
 false-positive errors if you use one in an annotation:
 
 ```py
-from typing_extensions import Final, ClassVar, Required, NotRequired, ReadOnly, TypedDict
```

Comment:
I removed `ClassVar` as "unsupported" here, but left the `invalid-base` test below, as it does not have an analog yet.